ticdc/manage-ticdc.md

```diff
@@ -178,18 +178,23 @@ The following are descriptions of parameters and parameter values that can be co
 | :------------------ | :------------------------------------------------------------ |
 | `127.0.0.1`          | The IP address of the downstream Kafka services                                 |
 | `9092`               | The port for the downstream Kafka                                          |
-| `topic-name`         | Variable. The name of the Kafka topic                                      |
-| `kafka-version`      | The version of the downstream Kafka. Optional, `2.4.0` by default. Currently, the earliest supported Kafka version is `0.11.0.2` and the latest one is `2.7.0`. This value needs to be consistent with the actual version of the downstream Kafka.                      |
-| `kafka-client-id`    | Specifies the Kafka client ID of the replication task. Optional. `TiCDC_sarama_producer_replication ID` by default. |
-| `partition-num`      | The number of the downstream Kafka partitions. Optional. The value must be **no greater than** the actual number of partitions. If you do not configure this parameter, the partition number is obtained automatically. |
-| `max-message-bytes`  | The maximum size of data that is sent to Kafka broker each time (optional, `64MB` by default) |
+| `topic-name` | Variable. The name of the Kafka topic |
+| `kafka-version`      | The version of the downstream Kafka (optional, `2.4.0` by default. Currently, the earliest supported Kafka version is `0.11.0.2` and the latest one is `2.7.0`. This value needs to be consistent with the actual version of the downstream Kafka)                      |
+| `kafka-client-id`    | Specifies the Kafka client ID of the replication task (optional. `TiCDC_sarama_producer_replication ID` by default) |
+| `partition-num`      | The number of the downstream Kafka partitions (optional. The value must be **no greater than** the actual number of partitions; otherwise, the replication task cannot be created successfully. `3` by default). Note that from v4.0.16, the default value has changed from `4` to  `3`. |
+| `max-message-bytes`  | The maximum size of data that is sent to Kafka broker each time (optional, `1MB` by default). Note that from v4.0.16, the default value has changed from `512MB` to  `1MB`. |
```

Comment:
```suggestion
| `partition-num`      | The number of the downstream Kafka partitions (optional. The value must be **no greater than** the actual number of partitions; otherwise, the replication task cannot be created successfully. `3` by default). Note that from v4.0.16, the default value has changed from `4` to `3`. |
| `max-message-bytes`  | The maximum size of data that is sent to Kafka broker each time (optional, `1MB` by default). Note that from v4.0.16, the default value has changed from `512MB` to `1MB`. |
```